tmt/steps/provision/artemis.py

```diff
@@ -290,9 +290,9 @@ class ProvisionArtemis(tmt.steps.provision.ProvisionPlugin):  # type: ignore[mis
 
     # TODO: fix types once superclass gains its annotations
     @classmethod
-    def options(cls, how: Any = None) -> List[click.Option]:
+    def options(cls, how: Any = None) -> List[tmt.options.ClickOptionDecoratorType]:
         """ Prepare command line options for Artemis """
-        return cast(List[click.Option], [
+        return cast(List[tmt.options.ClickOptionDecoratorType], [
```

Comment:
This `cast` and comment above isn't needed anymore.

Reply:
I'll leave it, not worth yet another round of rebuilds... We have a task to review all `cast` calls, it's time will come.

Reply:
Ok, I'm fine with that (if the current version is a merge candidate).